paper-input.html

```diff
@@ -120,9 +120,10 @@
 
       <content select="[prefix]"></content>
 
-      <label hidden$="[[!label]]" aria-hidden="true" for="input">[[label]]</label>
+      <label hidden$="[[!label]]" aria-hidden="true">[[label]]</label>
```

Comment:
woof, this is bad. This break a11y. @robdodson, halp, do you have any suggestions?

Reply:
We should probably just generate the ID using something like this: https://github.com/PolymerElements/paper-input/blob/master/paper-input-behavior.html#L21

Reply:
(and then manually add it to the label)